libmount/src/hook_loopdev.c

```diff
@@ -181,7 +183,7 @@ static int setup_loopdev(struct libmnt_context *cxt,
 	 * encryption=
 	 */
 	if (!rc && mnt_optlist_get_opt(ol, MNT_MS_ENCRYPTION, cxt->map_userspace)) {
-		DBG(LOOP, ul_debugobj(cxt, "encryption no longer supported"));
+		mnt_context_save_failure(cxt, hs, 0, EINVAL, MNT_MS_ENCRYPTION, NULL);
```

Comment:
Below both the debugging and `mnt_context_save_failure()` are kept.
Here the debugging is dropped. Is this intentional?